.github/workflows/build-and-push.yml

```diff
@@ -0,0 +1,27 @@
+name: Build to release branches
+
+on:
+    workflow_dispatch:
+    push:
+        tags: ['*']
+        branches:
+            - dev-beta
+            - dev-main
+
+concurrency:
+    group: ${{ github.workflow }}-${{ github.ref }}
+    cancel-in-progress: false
+
+jobs:
+    build-assets:
+        uses: inpsyde/reusable-workflows/.github/workflows/build-and-push-assets.yml@add_option_to_select_built_branch_name
+        with:
+            PHP_VERSION: '8.1'
+            PHP_TOOLS: composer:v2
+            BUILT_BRANCH_NAME: "${{ (github.ref_name == 'dev-main' && 'main' || (github.ref_name == 'dev-beta' && 'beta') || '') }}"
+            COMPILE_SCRIPT_PROD: 'build'
+            COMPILE_SCRIPT_DEV: 'build:dev'
```

Comment:
Since these values are the default (see https://github.com/inpsyde/reusable-workflows/blob/f01918b1ef12f2b748bed8d0796ff2f75577f695/.github/workflows/build-and-push-assets.yml#L31-L40), can we omit them?
```suggestion
```